coldfront/core/allocation/models.py

```diff
@@ -192,6 +234,13 @@ def get_attribute(self, name, expand=True, typed=True,
         return None
 
     def set_usage(self, name, value):
+        """
+        Params:
+            name (str): allocation attribute type whose usage to set
+            value (int): value to set usage to
```

Comment:
Looks like this should be a float. 

Also remove `# check value type` to get rid of this warning: 
```
WARNING  -  mkdocstrings_handlers: coldfront.core.allocation.models.Allocation.set_usage: Failed to get 'name: description' pair from '# check value type'
```